cli/setup.go

```diff
@@ -153,13 +155,17 @@ const (
 
 // ---------------------------- END Setup constants ----------------------------
 
-func getDefaultDeviceType() string {
-	hostName, err := ioutil.ReadFile("/etc/hostname")
+func getDefaultDeviceType(ctx *cli.Context) (devType string) {
+	devType, err := device.GetDeviceType(path.
+		Join(ctx.GlobalString("data"), "device_type"))
```

Comment:
Is this correct? TBH I don't know exactly what `ctx.GlobalString("data")` is, but I thought we can use `conf.DefaultDeviceTypeFile` (used already below at `saveConfigOptions`).

Reply:
It's the value from the `--data` flag which gives the path to the directory containing the `device_type` file. It defaults to the same value, I just figured it might be a "more correct" value in case the flag is set.

Reply:
Thanks for the clarification :+1: